documentation/modules/managing/con-consumer-config-properties.adoc

```diff
@@ -65,6 +72,21 @@ group.id=my-group-id <1>
 ----
 <1> Add a consumer to a consumer group using a group id.
 
+== Choosing the right partition assignment strategy
+
+Select an appropriate partition assignment strategy, which determines how Kafka topic partitions are distributed among consumer instances in a group.
```

Comment:
Should the actual classes be mentioned here by name? Without that nobody can configure it to anything.